### ISSUE TITLE
Joplin do not start after installing plugin ...joplin fatal error unknown key: plugin-io.github.personalizedrefrigerator.js-draw.disable-editor-fills-window

### ISSUE THREAD

**Issue by hublex** [laurent22/joplin#8143]
The text couldn't be copied.

## Environment

Joplin version: should be the latest, I can't find the info cause it is not open
Platform: Windows 10

-->

## Steps to reproduce

1. 
2. 
3. 

## Describe what you expected to happen

Joplin isn't starting anymore after I did install the following plugin:

https://github.com/sadmice/Joplin-Plugin-Text-Colorize
After instaalling the plugin I did change the shortcut from Shift+STRG+C to Shift+STRG+X
Than I did try to restart the software - here I just get the following error massage and Joplin to not start anymore

## Logfile

![image](https://user-images.githubusercontent.com/75634476/236803448-f94d52d5-e234-4e2c-98c2-d69bbc7c0c41.png)

**Comment by laurent22** [comment]
Could you try installing the "Freehand Drawing" plugin again to see if it fixes the issue? But that should not happen regardless

**Comment by hublex** [comment]
Thank you Laurent. Is there a possibility without opening Joplin? Because Joplin do not open.

**Comment by Daeraxa** [comment]
Try enabling safe mode - https://joplinapp.org/debugging/#safe-mode. That should allow you to start it. Otherwise you can find your plugins in your config folder and remove it from there (should be in `C:\Users\<username>\.config\joplin-desktop\plugins`).

Just because I mention it every time, make a backup of the whole `joplin-desktop` directory before messing around in there just in case something happens.

**Comment by laurent22** [comment]
Replicated (just copying error here for reference)

```
main-html.js:49 Error: Unknown key: plugin-io.github.personalizedrefrigerator.js-draw.disable-editor-fills-window
    at Setting.settingMetadata (Setting.ts:1813:33)
    at exports.default (mergeGlobalAndLocalSettings.ts:15:22)
    at Setting.<anonymous> (Setting.ts:1960:47)
    at Generator.next (<anonymous>)
    at fulfilled (/Users/laurent/src/joplin/packages/lib/models/Setting.js:5:58)
```

**Comment by laurent22** [comment]
If Daeraxa's suggestion doesn't work, what you could do for now is open the file `:\Users\<username>\.config\joplin-desktop\profiles.json` and here change the "currentProfileId" value to another profile ID. For example to "default". If it's not clear, please post your profiles.json file here

**Comment by personalizedrefrigerator** [comment]
For reference, [here's how js-draw is registering settings](https://github.com/personalizedrefrigerator/joplin-plugin-freehand-drawing/blob/f012f0d1ee2bd3aa558b896cfb287c0da712d116/src/index.ts#L65).

**Comment by laurent22** [comment]
> For reference, [here's how js-draw is registering settings](https://github.com/personalizedrefrigerator/joplin-plugin-freehand-drawing/blob/f012f0d1ee2bd3aa558b896cfb287c0da712d116/src/index.ts#L65).

Thanks, I noticed this. What happened is that most plugins don't register their settings to the filesystem, even though it's valid to do so. So this case has never been tested along with the multiple profiles feature. I'm going to release a new version with a hotfix for this and a few other minor issues.

**Comment by laurent22** [comment]
@hublex, does it work with the latest version? https://github.com/laurent22/joplin/releases/tag/v2.10.17

**Comment by hublex** [comment]
thx for reply, no it doesn't.
![image](https://user-images.githubusercontent.com/75634476/236905188-5d3e0532-7beb-47b3-bebf-1c2c59faf0ab.png)
Chenging the profile did, after I could update to the new version. I did delete the adon you have mantioned and also the one I have installed today.
I think I want need this but I couldn't fuigered it out how to use the safe mode https://joplinapp.org/debugging/#safe-mode
